tests/test_bank/test_eventbank.py

```diff
@@ -91,6 +91,11 @@ def ebank_with_bad_files(tmpdir):
     st.write(str(stream_path / "not_an_event.xml"), "mseed")
     bank = EventBank(stream_path)
     # should issue warning
+
+    # bank.index_path.unlink()
+    # breakpoint()
+    # bank.update_index()
+
```

Comment:
Is there a reason these commented lines are in here?

Reply:
I am going to chalk this up to coding while very tired ;)